pkg/server/server.go

```diff
@@ -406,7 +406,10 @@ func (s *Server) reportUploadCompleted(w io.Writer) {
 func (s *Server) backup(backupDirectoryID string, policyID string, progressOutput io.Writer) error {
 	ctx := context.Background()
 	// Create recovery point
-	rp, err := s.backupClient.CreateRecoveryPoint(ctx, backupDirectoryID, &backupapi.CreateRecoveryPointRequest{PolicyID: policyID})
+	rp, err := s.backupClient.CreateRecoveryPoint(ctx, backupDirectoryID, &backupapi.CreateRecoveryPointRequest{
+		PolicyID: policyID,
+		Name:     "auto-" + time.Now().Format(time.RFC3339),
```

Comment:
I think we should put name of recovery to `backup` function.
In case manual backup, the name of the recovery point is received from the broker message.

Add name field to here as well - https://github.com/bizflycloud/bizfly-backup/blob/master/pkg/broker/message.go#L34